documentation/api/api-poi.yaml

```diff
@@ -0,0 +1,284 @@
+openapi: 3.0.3
+info:
+  title: POI REST API
+  description: This document specifies the REST API for POI (proof of inclusion) in Chronicle.
+  contact:
+    email: contact@iota.org
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 2.0.0
+externalDocs:
+  description: Find out more about IOTA
+  url: https://iota.org
+servers:
+  - url: http://127.0.0.1:14265
+tags:
+  - name: create
+    description: Proof-of-Inclusion creation.
+  - name: validate
+    description: Proof-of-Inclusion validation.
+paths:
+  /api/poi/v1/create/{blockId}:
```

Comment:
Perhaps it would be better to use `/api/proof/...`?

Reply:
I think he's following the definition of `inx-poi` here.

Reply:
yeah, that's correct and it would be confusing for people if we use a different routes than they do.